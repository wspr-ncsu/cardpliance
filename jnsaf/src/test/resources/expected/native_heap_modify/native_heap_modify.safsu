`Lorg/arguslab/native_heap_modify/MainActivity;.heapModify:(Landroid/content/Context;Lorg/arguslab/native_heap_modify/Data;)V`:
  arg:2.str = java.lang.String@~
;