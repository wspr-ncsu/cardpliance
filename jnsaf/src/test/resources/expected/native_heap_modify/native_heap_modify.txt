Lorg/arguslab/native_heap_modify/MainActivity;.heapModify:(Landroid/content/Context;Lorg/arguslab/native_heap_modify/Data;)V -> _SOURCE_ 2.str